### PR TITLE
chore(setup): Change default SW360 repository to Eclipse

### PR DESCRIPTION
**Pull request by maierthomas**
- changes the default repository from sw360portal to eclipse/sw360
- bugfix regarding `git pull` (needs at least user configuration name and e-mail)

closes sw360/sw360vagrant#19

### CODE REVIEW

File: shared/configuration.rb [hunk 1]
```diff
@@ -48,8 +48,8 @@
 # For synchronization you might have to start `vagrant rsync-auto` manually
 SW360_source=""
 
-SW360_gitURL="https://github.com/sw360/sw360portal.git"
-SW360_branch="" # the value "" means: "don't change the branch"
+SW360_gitURL="https://github.com/eclipse/sw360.git"
+SW360_branch="master" # please specify the branch name of sw360 repository
```
Comment (alexbrdn) [inline shared/configuration.rb]: empty default was good enough, I think

File: sw360-single/sw360-install.sh [hunk 2]
```diff
@@ -138,14 +138,16 @@ if [ -z $SW360_source ]; then
...
+            git reset --hard origin/$SW360_branch
         else
             git clone -b $SW360_branch --single-branch $SW360_gitURL "$wd"
         fi
     else
         if [ -d "$wd/.git/" ]; then
             cd "$wd"
-            git pull $SW360_gitURL
+            git fetch
+            git reset --hard origin/master
```
Comment (alexbrdn) [inline sw360-single/sw360-install.sh]: please do not reset any current branch to `origin/master` , but rather to `origin/{current_branch}`

File: shared/configuration.rb (hunk 1 above)
Comment (maierthomas) [reply shared/configuration.rb]: removed master in SW360_branch

File: sw360-single/sw360-install.sh (hunk 2 above)
Comment (maierthomas) [reply sw360-single/sw360-install.sh]: Added new variable `currentBranch` and set value to `git rev-parse --abbrev-ref HEAD`

**Pull request merged**